ng2-components/ng2-activiti-processlist/src/components/activiti-processlist.component.ts

```diff
@@ -62,6 +62,7 @@ export class ActivitiProcessInstanceListComponent implements OnChanges, AfterCon
     onError: EventEmitter<any> = new EventEmitter<any>();
 
     currentInstanceId: string;
+    loadingFlag: boolean = true;
```

Comment:
you don't need a method isLoading that return loadingFlag. Or you create a get that return isLoading or you just use the property